assets/js/app/test_results.js

```diff
@@ -0,0 +1,244 @@
+// Stores the projects
+var project_urls = [
+  {
+    name: "4.4",
+    squad_url: "https://qa-reports.linaro.org/lkft/linux-stable-rc-4.4-oe/",
+    project_url: "https://qa-reports.linaro.org/api/projects/40/"
```

Comment:
Oh, please don't hardcode these here. We have _data/builds.yml. If it's missing a field, add it there please. They get updated regularly.